### PR TITLE
Added routes

### PR DESCRIPTION
**Pull request by sohan9819**
Product detail page :+1:
error 404 page :+1:

### CODE REVIEW

File: src/pages/ProductDetail.jsx [hunk 1]
```diff
@@ -8,3 +8,46 @@
...
+
+  const getProductDetails = (products, productId) => {
+    return products.find((product) => product._id === productId);
+  };
+
+  const [product, setProduct] = useState({});
+
+  useEffect(() => {
+    // console.log(products);
+    // console.log(productId);
```
Comment (Balakumar3018) [inline src/pages/ProductDetail.jsx:31]: remove commented codes

**Review by Balakumar3018 — APPROVED** [review]
Good to merger.

File: src/pages/Cart.jsx [hunk 2]
```diff
@@ -21,7 +22,10 @@ const Cart = () => {
           <CartProducts>
             {cartItems.length === 0 ? (
               <h1 className='h1'>
-                No items in the cart <br /> 🙈🙉🙊
+                No items in the cart <br /> 🙈🙉🙊 <br />
```
Comment (athar05) [inline src/pages/Cart.jsx:25]: I think Tanay mentioned that br tags shouldn't be used with JSX. I am not sure what the replacement for this is but you can consider looking into this.

**Review by athar05 — COMMENTED** [review]
All the routes seem to be working fine. The UI also looks amazing. 
I think the PR can be merged.

**Review by rahul31agrawal — APPROVED** [review]
Hi Sohan,

Looks good to merge

Thanks

**Pull request merged**